django-stubs/db/backends/base/client.pyi

```diff
@@ -4,6 +4,6 @@ from django.db.backends.base.base import BaseDatabaseWrapper
 
 class BaseDatabaseClient:
     executable_name: Any = ...
-    connection: Any = ...
+    connection: BaseDatabaseWrapper
```

Comment:
Why not `connection: BaseDatabaseWrapper = ...`?

Reply:
To me the ` = ...` seems to imply that this attribute has a value at class-definition-time. But the `connection` attribute only exists on instances, not the class itself.

But I don't feel strongly about this, I can add the `...` back, what do you think?